Gordon360/Services/ServiceInterfaces.cs

```diff
@@ -297,4 +302,94 @@ public interface IAcademicCheckInService
         Task<bool> GetStatusAsync(string id);
     }
 
+    namespace RecIM
+    {
+        public interface IActivityService
+        {
+            IEnumerable<LookupViewModel> GetActivityLookup(string type);
+            IEnumerable<Models.ViewModels.RecIM.ActivityExtendedViewModel> GetActivities();
```

Comment:
With your most recent set of updates to the API, I don't even think we need to explicitly state RecIM ActivityViewModel since the involvement activity vm has changed.

Reply:
Didn't even need the specifier at all